src/rpc/rpccategory.h

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2010 Satoshi Nakamoto
+// Copyright (c) 2009-2018 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_RPC_RPCCATEGORY_H
+#define BITCOIN_RPC_RPCCATEGORY_H
+
+#include <string>
+
+enum class RPCCategory
+{
+    blockchain = 0,
+    control,
+    generating,
+    hidden,
+    mining,
+    network,
+    rawtransactions,
+    util,
+    wallet,
+    zmq,
+    test
+};
+
+namespace rpccategory
+{
+
+extern std::string Label (const RPCCategory category);
```

Comment:
remove extern

Reply:
hm… `extern` (ok I'm old style C developer) means `declared elsewhere`, infact the prototype is declared in `src/rpc/rpccategory.cpp`, and the definition also in `src/rpc/rpccategory.h`. What's wrong?

Reply:
`extern` on function is redundant, it is already `extern` if you don't implement it.